kingfisher_scrapy/spiders/peru_compras_bulk.py

```diff
@@ -0,0 +1,44 @@
+import scrapy
+
+from kingfisher_scrapy.spiders.peru_compras_base import PeruComprasBase
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class PeruComprasBulk(PeruComprasBase):
+    """
+    Domain
+      Peru Compras (contracts within framework agreements)
+    Caveats
+        The JSON data sometimes contains unescaped newline characters within strings.
+        The peru_compras spider contains more updated data.
+    Spider arguments
+      from_date
+        Download only data from this month onward (YYYY-MM format). Defaults to '2021-08'.
+      until_date
+        Download only data until this month (YYYY-MM format). Defaults to the current month.
+    """
+
+    name = 'peru_compras_bulk'
+
+    # BaseSpider
+    date_format = 'year-month'
+    default_from_date = '2021-08'
+
+    def start_requests(self):
+        yield scrapy.Request(
+            f'{self.url_prefix}getListaDescargaMasiva?Anio=&Mes=',
+            method='POST',
+            meta={'file_name': 'list.json'},
+            callback=self.parse_list
+        )
```

Comment:
To make it easy to compare the two spiders:

```suggestion
        url = f'{self.url_prefix}getListaDescargaMasiva?Anio=&Mes='
        yield scrapy.Request(url, method='POST', meta={'file_name': 'list.json'}, callback=self.parse_list)
```